features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioBLE.cpp

```diff
@@ -392,17 +392,12 @@ void BLE::callDispatcher()
     // process the external event queue
     _event_queue.process();
 
-    // follow by stack events
-    static us_timestamp_t last_time_us = _timer.read_high_resolution_us();
-
-    // Update the current cordio time
-    us_timestamp_t curr_time_us = _timer.read_high_resolution_us();
-    uint64_t delta_time_ms = ((curr_time_us - last_time_us) / 1000);
+    int delta_time_ms = _timer.read_ms();
 
     if (delta_time_ms > 0) {
```

Comment:
shouldn't this check (delta_time_ms / WSF_MS_PER_TICK) is greater than 0 instead? do we want to call WsfTimerUpdate with 0?